RecoBTag/DeepFlavour/src/sorting_modules.cpp

```diff
@@ -0,0 +1,29 @@
+
+
+
+#include "../interface/sorting_modules.h"
+#include <iostream>
+namespace sorting{
+
+std::vector<std::size_t> invertSortingVector(const std::vector<sortingClass<std::size_t> > & in){
+    std::size_t max=0;
+    for(const auto& s:in){
+        if(s.get()>max)max=s.get();
+    }
+
+    if(max>1e3){
+        for(const auto& s:in){
+          std::cout << s.get() << std::endl;
```

Comment:
how long is this printout if it happens?
Only a brief message is meaningful by default, long dumps could be left for debugging.

This part should still be made into something thread safe so that the message is not garbled in multithread running. Use message logger or push to a string or a stream to be dumped together with the message.

Reply:
I think that debug message is not needed, even if the vector is very large the actual values do not give you much information for debuggin. Maybe @jkieseler can provide further info on the reason for this.

Reply:
removed with https://github.com/cms-sw/cmssw/pull/19893/commits/6ad48f2a3527cf49f3a753cf57bfe30380048472